SCR/TestRunner.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+from spike_partials.test_RecordTarget import RecordTargetTest
+from spike_partials.test_ReplacementOf import ReplacementOfTest
+
+if __name__ == '__main__':
+    #suite = unittest.TestLoader().loadTestsFromTestCase(RecordTargetTest)
+    #unittest.TextTestRunner(verbosity=2).run(suite)
```

Comment:
Are these comments still required?

Reply:
I've removed this file as it was mainly concerned with the spike tests I build when investigating mustache

Reply:
👍